docker/images/skycoindev-cli/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/* 
 
 # Installs Criterion
-RUN git clone --recurse-submodules -j8 https://github.com/Snaipe/Criterion && \
+RUN git clone --recurse-submodules -j8 https://github.com/mlopezqc/Criterion && \
```

Comment:
Why the change?

Reply:
Error after a change in github.com/Snaipe/Criterion see #1481 . I am of the opinion that we should continue using the same repository but checkout an specific version (hash , since this is in `bleeding`)

Reply:
... but a separate repository could be ok . @stdevMauricio if this is the way to go then change the repos to be http://github.com/simelo/Criterion .

Reply:
Both suggestions done.